info/available-rules.md

```diff
@@ -1,131 +1,131 @@
+| 1    | 1.1.1    | IDENTIFIER_LENGTH                         | Check: identifier length should be in range [2,64] except names that are used in industry like {i, j} and 'e' for catching exceptions.<br>Fix: no fix is available to automatically create a name                                                         | no  | no                                                                                                                                                                                                                         | May be make this rule configurable (length)                                                                                                                                                                                   |
+| 1    | 1.1.1    | FILE_NAME_INCORRECT                       | Check: warns if a file name does not have an extension .kt/.kts<br>Fix: no                                                                                                                                                                                        | no  | no                                                                                                                                                                                                                         | Extensions should be configurable; it can be autofixed                                                                                                                                                             |
+| 1    | 1.1.1    | GENERIC_NAME                              | Check: warns if a generic name contains more than 1 letter (capital). It can be followed by numbers, for example: T12, T<br>Fix:                                                                                                                                   | yes | no                                                                                                                                                                                                                         | Recursively update usages of this identifier in the project                                                                                                                                                                   |
```

Comment:
> Check: warns if **the** name of **a** [_generic type parameter_](https://kotlinlang.org/docs/generics.html) (e.g: `T`) consists of more than a single capital letter. The capital letter can be followed by numbers, though, e. g.: `T12`.

Reply:
Check: warns if the name of a  [_generic type parameter_](https://kotlinlang.org/docs/generics.html) (e.g. `T`) consists of more than a single capital letter. The capital letter can be followed by numbers, though (e. g. T12), T

Reply:
>  (e. g. T12, T).

No, this example is misleading now.

`T12` is a good example, `T` is not (not followsed by numbers).

Also, please don't forget to surround code fragments with backticks, e.g.: `T12`.

Reply:
OK